packages/pipeline/src/parsers/radar_orders/index.ts

```diff
@@ -0,0 +1,121 @@
+import { ObjectMap } from '@0x/types';
+import { BigNumber } from '@0x/utils';
+import { RadarBook, RadarMarket, RadarSignedOrder } from '@radarrelay/types';
+import * as R from 'ramda';
+
+import { aggregateOrders, GenericRawOrder } from '../utils';
+
+import { TokenOrderbookSnapshot as TokenOrder } from '../../entities';
+import { OrderType } from '../../types';
+
+export interface AggregateOrdersByMaker {
+    makerAddress: string;
+    price: string;
+    amount: BigNumber;
+}
+
+/**
+ * Marque function of this file.
```

Comment:
Nit: should be spelled `marquee`. :P

Reply:
hah they all have this problem. fixing them.